### PR TITLE
doc: prepare new release 3.8.0

### PR DESCRIPTION
**Pull request by PhilippeAccorsi**
Related to #4429 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- No manually test necessary

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+- Add and comment a publication
+- Home/login page can be customizable 
+- Preview of links can be genearated in comment
+- Be able to fold the subspaces in the sidebar
+- New space dashboard
+- Possibilities to modify comments
+- New navigation links to user profile
+- Add button "open the gallery" on the content page
+- Allow to open file uploaded in a comment as a content
+- Rich text available in dashboard description
```
Comment (raphj) [inline changelog.md:15]:
```suggestion
- Content can be added to personal favourites
- Add and comment a publication
- The Home/login page can be customized 
- Links in comments are now previewed when possible
- Subspaces can be folded in the sidebar
- the dashboard has been revamped
- Comments can be edited and removed
- User profile can now be reached from places where the user has posted something 
- There is a new button on the content page to open the gallery
- files uploaded in a comment can now be opened as a content
- The space description is now editable using a rich text editor
```

Comment (gbonaspetti) [reply changelog.md:15]: @raphj I update your suggestion with some details (to not have multiples suggestions for the same thing)

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+
+### Fixed Issues
+[#](https://github.com/tracim/tracim/issues/)
```
Comment (gbonaspetti) [inline changelog.md:18]: is this "generic" line normal?

File: changelog.md (hunk 1 above)
Comment (gbonaspetti) [reply changelog.md:15]: @raphj I update your suggestion with some details (to not have multiples suggestions for the same thing)

Comment (raphj) [reply changelog.md:15]: good call! :-)

File: changelog.md (hunk 2 above)
Comment (PhilippeAccorsi) [reply changelog.md:18]: no :(

File: changelog.md (hunk 1 above)
Comment (PhilippeAccorsi) [reply changelog.md:15]: done

File: changelog.md (hunk 2 above)
Comment (PhilippeAccorsi) [reply changelog.md:18]: Done

Comment (gbonaspetti) [reply changelog.md:18]: ok, so I little suggestion to apply :smile: 
```suggestion
```

**Review by gbonaspetti — APPROVED** [review]

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,36 @@
...
+Frontend: [#3787](https://github.com/tracim/tracim/issues/3787),
+[#3811](https://github.com/tracim/tracim/issues/3811),
+[#4464](https://github.com/tracim/tracim/issues/4464),
+[#4532](https://github.com/tracim/tracim/issues/4532),
+[#4567](https://github.com/tracim/tracim/issues/4567)
+Backend: [#4552](https://github.com/tracim/tracim/issues/4552)
+
+### Breaking/Important change
+
+- Renaming some wording in Tracim (issue [#4348](https://github.com/tracim/tracim/issues/4348)):
```
Comment (raphj) [inline changelog.md:28]:
```suggestion
- Some rewording in Tracim (issue [#4348](https://github.com/tracim/tracim/issues/4348)):
```

**Review by raphj — APPROVED** [review]

File: changelog.md [hunk 4]
```diff
@@ -1,3 +1,36 @@
+# 3.8.0 / 2021-04-27
+
+### New Features
+
+- Content can be added to personal favourites
+- Add and comment a publication
```
Comment (grignards) [inline changelog.md:6]: Is this feature the publications by themselves? Publications were added in the 3.7.0 I think.

File: changelog.md [hunk 5]
```diff
@@ -1,3 +1,36 @@
+# 3.8.0 / 2021-04-27
+
+### New Features
+
+- Content can be added to personal favourites
```
Comment (grignards) [inline changelog.md:5]:
```suggestion
- Contents can be added to personal favourites
```

File: changelog.md (hunk 3 above)
Comment (PhilippeAccorsi) [reply changelog.md:28]: Done

File: changelog.md (hunk 5 above)
Comment (PhilippeAccorsi) [reply changelog.md:5]: Done

File: changelog.md (hunk 4 above)
Comment (PhilippeAccorsi) [reply changelog.md:6]: thats right. its a mistake in project 3.8.0 description.
I remove this part.

Comment (PhilippeAccorsi) [reply changelog.md]: done

**Review by raphj — APPROVED** [review]

**Review by grignards — APPROVED** [review]

**Pull request merged**